WebContent/html/admin/nutzeraktualisieren.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
```

Comment:
Hallo Merlin,
ich würde die Administrationsseiten in den WEB-INF Ordner verschieben. Bei dieser Struktur können die Administrationsseiten über Direkt-URLs erreicht werden (hat uns der Stiehl in der Vorlesung gezeigt). Im WEB-INF Ordner können die Seiten nur über Servlets erreicht werden.